common/src/main/kotlin/com/hartwig/actin/trial/input/FunctionInputResolver.kt

```diff
@@ -929,6 +940,18 @@ class FunctionInputResolver(
         }
     }
 
+    private fun toTnmTs(input: Any): Set<TnmT>{
+        return toStringList(input).map(::toTnmT).toSet()
```

Comment:
I think you can do this without `toTnmT`:
```suggestion
        return toStringList(input).map(TnmT::valueOf).toSet()
```